app/settings/data-export/data-export.html

```diff
@@ -27,10 +27,36 @@ <h1 class="mode-context-title" translate>app.import</h1>
         <div class="main-col">
             <div class="form-sheet" ng-show="!showFields">
                 <div class="form-sheet-summary">
-                    <h2 class="form-sheet-title" translate>
-                    data_export.title
+                    <h2 class="form-sheet-title" ng-show="!hxlEnabled" translate>
+                     data_export.title
+                    </h2>
+                    <h2 class="form-sheet-title" ng-show="hxlEnabled" translate>
+                     data_export.title_hxl
```

Comment:
I think this duplication is unnecessary, let's combine them and simply pass a var {{ data_export_title }}